tests/tests.cpp

```diff
@@ -49,6 +51,102 @@ TEST(Serialize64, serialize) {
     ASSERT_EQ(in, out);
 }
 
+// Test encryption/decryption using AES in mode GCM
+// (test vectors from 'tlvstore_test.js', in Webclient)
+TEST(CryptoPP, AES_GCM)
+{
+//    string keyStr = "dGQhii+B7+eLLHRiOA690w==";   // Base64
```

Comment:
It's a reminder to remark why the key is not the same than in `tlvstore_test.js`, where Base64URL encoding is not used, as it happens in the SDK. I'd prefer to keep it if you don't mind.